paymentsheet/src/main/java/com/stripe/android/paymentsheet/ExternalPaymentMethodConfirmHandler.kt

```diff
@@ -11,6 +11,7 @@ interface ExternalPaymentMethodConfirmHandler {
     fun createIntent(
         context: Context,
         externalPaymentMethodType: String,
+        billingDetails: PaymentSheet.BillingDetails,
```

Comment:
I originally didn't do this, because the iOS API has billing details as a non-nillable field and wanted to be consistent with that. I just made this change because it's simpler to do for our implementation, but probably something we should talk about during API review. 

It doesn't seem like it would make a big difference for merchants to have this be nullable or not, since they would need to handle individual fields being nullable even if billing details itself is not null. And I'm not sure how much we want to be consistent with iOS here (probably doesn't matter a ton, given that the rest of the API will likely be pretty different).